pkgs/applications/system/tango-controls/default.nix

```diff
@@ -0,0 +1,101 @@
+{ cmake
+, cppzmq
+, doxygen
+, fetchurl
+, graphviz
+, lib
+, libjpeg_turbo
+, libsodium
+, mariadb-connector-c
+, openjdk11
+, pkg-config
+, python3
+, stdenv
+, systemd
+, zeromq
+, zlib
+}:
+let
+  # omniorb is in nixpkgs, but tango doesn't support the latest version
+  omniorb_4_2 = stdenv.mkDerivation rec {
```

Comment:
How about overriding?

Reply:
Yep, shortens the derivation a bit, fixed!